content/work/adobe.md

```diff
@@ -29,14 +64,313 @@ press:
+font/illustration:
+
+```js
+fontStyles() {
+  return {
+    '--svg-adjust': this.font.adjust,
+  };
+}
+```
+
+{{ embed.figure(
+  data=compare,
+  caption='The example on the left shows the type extending outside it’s
+    container and the right example is what this illustration looked like
+    after we adjusted the font-size',
+  class='extend-small size-quarter'
+) }}
+
+This method was easily replicated across the entire project and allowed us to
+have more control over the design, while writing less code.
+
+{{ embed.figure(
+  data=after,
+  caption='After applying adjustments to only the fonts that needed it,
+    the result shows visually similar type size.'
+) }}
+
+### SVG Animation
+
+We were privileged to have Christina 
+join us for this project.
+Christina worked with our designers 
+to create clean and optimized SVGs foranimating. 
+She used the [GreenSock animation library] 
+to create varied animations for each font question. 
+Each font had a distinct personality 
+she built into the animation. 
+She worked with the whole team and Adobe 
+to adjust timing and easing on animations. 
+GreenSock made it easier to iterate on animations, 
+helping us to meet the project deadline.
+
+[GreenSock animation library]: https://greensock.com/
+
+{{ embed.codepen(
+  id='rNedLbN',
+  title='Relax Question',
+  user='cgorton'
+) }}
+
+### Usability Testing
+
+Testing web applications with users for the first time is exciting and a bit
+nerve-racking. Are we headed in the right direction? We tested the Font Tags
+Game with users as early in the process as possible – as soon as we had the
+questions, options, and animations in place, but before adding sounds and
+superpowers. Since our timeline was tight, we ran half-hour sessions via video
+with two volunteer designers. Their feedback was invaluable. They caught
+typeface alignment issues, pointed out confusing language, and helped us imagine
+new features for phase two. Most importantly, they let us know we were hitting
+our goals, describing their feelings afterwards as curious, optimistic,
+empowered, and ready to explore typography!
+
+{{ embed.figure(
+  data=herolink,
+  caption='Two testers helped us discover that the “link to heroes” was
+    distracting them from playing the game.'
+) }}
+
+### Launch
+
+Adobe launched the game in October 2020. 
+They were able to begin collecting and analyzing data 
+from users via Google Analytics right away. 
+Visitors reported a fun experience 
+and shared their typographic superpowers on social media. 
+
+"Adobe you see right through me! 'Being too confrontational' 😂 Such a fun game. Check it out!" - @StatUXofLiberty
+
+"okay this was actually very cool and fun. good job, adobe 😂 I found my typographic superpower. What's yours?" - @manic_femme
+
+"I got... Funky!! 🤣✨ Of course these lyrics come to mind: 🎵 Shining through the city with a little funk and soul / So I'ma light it up like dynamite, whoa oh oh 🎵" - @froznberry
```

Comment:
I think these should either be markdown blockquotes, or use our `quote.grid()` macro. @SondraE you can see examples of both in the `/sample/type.md` file, and see what they look like at https://www.oddbird.net/sample/type/#quotes (feel free to ping me if that doesn't make sense)

Reply:
Oh, I converted them to blockquotes before I saw this note. Feel free to use `quote.grid` if you prefer.

Reply:
Blockquotes look best to me.